results/proto/api.proto

```diff
@@ -20,6 +21,7 @@ message CreateTaskRunRequest {
 
 message UpdateTaskRunRequest {
 	TaskRun task_run = 1;
+	google.protobuf.FieldMask update_mask = 2;
```

Comment:
Proto files are often used as the API definition itself. It doesn't look like we're doing anything with this field, so we should remove this for the time being.

Reply:
Do you mean it is okay for now not to follow api definition in Google AIP guideline?

Reply:
Discussed offline- the ultimate goal is to support this, but for now it doesn't make much sense to have a field that isn't actually used.

Reply:
Done